src/main/resources/templates/fragments/usersFragment.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
```

Comment:
I'm not sure how well IntelliJ supports prettier, but I'd recommend using prettier for formatting HTML, CSS and JS. Just keeps things consistent. In VSCode I have it set to apply the formatting on save.

Reply:
Thanks! I haven't installed it because it seems like the same thing as Reformat File in Intellij. Finding some way to do it automatically would probably be a good idea, but I want to turn off one setting that adds new line breaks if the line is too long. I find it much easier to work with very long lines in some circumstances like an HTML element with many classes. I rarely need to see all of them and usually I'd rather just do a find/replace or scroll around.